src/langcheck/metrics/eval_clients/_llama.py

```diff
@@ -66,10 +75,22 @@ def __init__(
             "ja": "あなたは誠実で優秀な日本人のアシスタントです。以下は、タスクを説明する指示です。要求を適切に満たす応答を日本語で書きなさい。",
         }
 
+        if extractor is None:
+            self._extractor = LlamaExtractor(
+                model=self._model,
+                tokenizer=self._tokenizer,
+                sampling_params=self._sampling_params,
+                system_prompt=system_prompt,
+            )
+        else:
+            self._extractor = extractor
+
     def get_text_responses(
         self,
         prompts: list[str],
         language: str,
+        *,
+        tqdm_description: str | None = None,
```

Comment:
Note: This should resolve https://github.com/citadel-ai/langcheck/issues/181, but there seems no custom tqdm description available in vllm so this will be just ignored.

Reply:
Thanks! As far as it resolves the runtime error it's fine for now.